component-documentation/templates/componentGroup.template.html

```diff
@@ -1,63 +1,37 @@
-{# Defines macros for reusable templates. #}
-{% macro method(method) -%}
-  {% include 'method.template.html' %}
-{% endmacro %}
-
-{% macro property(property) -%}
-  {% include 'property.template.html' %}
-{% endmacro %}
-
-{% macro methodList(methodList) -%}
-  {% include 'method-list.template.html' %}
-{% endmacro %}
-
-{% macro propertyList(propertyList) -%}
-  {% include 'property-list.template.html' %}
-{% endmacro %}
-
-{% macro class(class) -%}
-  {% include 'class.template.html' %}
-{% endmacro %}
+{# Defines macros for reusable templates. #} {% macro method(method) -%} {% include 'method.template.html' %} {%
```

Comment:
das ist zwar kürzer aber nicht wirklich prettier XD

Reply:
Da es leider als html geparsed wird, wird das alles als Text interpretiert und der kommt in eine Zeile 🤷 Weißt du, ob die html-Endung nötig ist? Alternativ kann ich schauen, dass ich für `template.html` Dateien prettier deaktiviere

Reply:
Von mir aus können wir den ganzen Ordner "component-documentation" auch wegschmeißen^^

Reply:
Wenn der nicht mehr gebraucht wird, sollte er wirklich weg 🗑️